### DISCUSSION TITLE
Solved: Recent releases disable IDEX, force "filament changes" instead

### DISCUSSION THREAD

**Discussion by Tickolas** [JGMaker3dofficial/artistd#53] [Q&A] [answered]
Hiya!
So I've tried the latest RC and beta release, and they both make my Artist-D do a filament change routine when it should be changing from E0 to E1.  The same gcode worked fine with IDEX when I was running the 1.1.4 Beta. 
Am I doing something wrong here, or is there a bug? Do I need a new Cura profile or something?
Sample gcode attached.
[calibration.gcode.txt](https://github.com/JGMaker3dofficial/artistd/files/6177305/calibration.gcode.txt)

Going back to 1.1.4 for now, but thanks so much for your great work guys. Truly appreciate you =D

**Comment by Tickolas** [comment, marked as answer]
Answer: The filament sensors need to be disabled. Again. D'oh! The compliments still stand ^^

**Reply by mikejr83** [reply]
If you turn off the run out sensors and then turn the printer off without saving it will re-enable them. I'm pretty sure if you go to configuration and store settings once you've turned them off that will persist. However, do not rely on this to hold true after a firmware flash. It's generally a best practice to re-initialize the settings from Configuration ->Advanced Settings -> Initialize EEPROM after a flash and then re-apply your settings. This is suggested as if we enable new functionality that has new settings it can get some wonky default values after the flash.

**Comment by TwinkieXLII** [comment]
That's really weird. I use the runout sensors and don't have that issue.

**Comment by Tickolas** [comment]
Well, I have them disconnected. So when I updated the firmware they were 
reenabled which made the machine think the filament ran out constantly 😃
